ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -359,7 +361,7 @@ - (RCTBridge *)bridge {
 
 RCT_REMAP_METHOD(displayMessage,
                  messageId:(NSString *)messageId
-                 overlay:(BOOL *)overlay
```

Comment:
note: these methods previously had BOOL pointers in the signature, which I have to assume was a mistake.